swap_meet/clothing.py

```diff
@@ -0,0 +1,14 @@
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "", condition = 0):
```

Comment:
Since you don't use `category` in the body of your function you should remove it from the parameters:

```suggestion
    def __init__(self, condition = 0):
```